compiler/one-cmds/tests/one-import_005.test

```diff
@@ -31,7 +33,7 @@ outputfile="test_onnx_model.circle"
 rm -f ${outputfile}
 
 # run test
-one-build -C ${configfile} > ${filename}.log 2>&1
+one-import onnx -C ${configfile} > ${filename}.log 2>&1
```

Comment:
it would be better not to change this but add new test

Reply:
okay I will add new test case.

Reply:
sorry for my mistake... current change is OK.